src/components/hv-switch/index.js

```diff
@@ -61,14 +61,24 @@ export default class HvSwitch extends PureComponent<HvComponentProps> {
         newElement.setAttribute('value', value ? 'on' : 'off');
         this.props.onUpdate(null, 'swap', this.props.element, { newElement });
       },
-      thumbColor: unselectedStyle ? unselectedStyle.color : null,
+      thumbColor: unselectedStyle?.color || selectedStyle?.color,
       trackColor: {
         false: unselectedStyle ? unselectedStyle.backgroundColor : null,
         true: selectedStyle ? selectedStyle.backgroundColor : null,
       },
       value: this.props.element.getAttribute('value') === 'on',
     };
 
+    if (Platform.OS === 'android' && !props.thumbColor) {
+      props.thumbColor = props.value ? props.trackColor.true : '#FFFFFF';
+    }
```

Comment:
Shouldn't this logic come after the two if conditions that set `props.thumbColor` below?

Reply:
* By default (above), on iOS, we leave `thumbColor` as `undefined` 
* Otherwise (here) we defaulting the `thumbColor` to the `trackColor` on android when selected, and white when not selected, to match the normal appearance of Android switches
* Then the two `if` conditions afterwards allows this two defaults to be overridden if set specifically.

Let me know if that makes sense.

Reply:
Got it, makes sense